src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../typedefs/Comment';
+
+interface Props {
+  postComments: Comment[]
+}
+
+export const CommentList: React.FC<Props> = ({ postComments }) => (
+  <ul className="comment-list">
+    {postComments.map(comment => (
```

Comment:
```suggestion
    {postComments.map(({ id, name, body, email }) => (
```

Reply:
why?

Reply:
Cause we can see fields that comment include and we can pass each of them to CommentInfo component.

Reply:
if we want to see what fields a particular type or an interface includes, we can check the interface itself. destructuring unnecessary props here would be similar to declaring a variable and not using it